packages/flutter/lib/src/material/checkbox.dart

```diff
@@ -714,7 +749,12 @@ class _CheckboxPainter extends ToggleablePainter {
     return Paint()
       ..color = checkColor
       ..style = PaintingStyle.stroke
-      ..strokeWidth = _kStrokeWidth;
+      ..strokeWidth = _isCupertino
```

Comment:
Feel like `_isCupertino` is not necessary if we already have the enum. Can we do something like this? 
```
..StrokeWidth = switch (designSpec) {
  _DesignSpec.cupertino => _kCupertinoStrokeWidth,
  _DesignSpec.material => _kStrokeWidth,
}
```
Here and all the if (_isCupertino) statements below:)